### DISCUSSION TITLE
part-ii/2017-2/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#182] [Question comments]
Part II 2017 Algebraic Geometry: Paper 4, Section II, I 

https://questions.tripos.org/part-ii/2017-2/

**Comment by silkca** [comment]
The first part is all bookwork.

For the second part it is far simpler to use the shorthand forms of degree and ramification as relating to numbers and multiplicities of preimages. Some care should be taken with the point at infinity in $\mathbb{P}^{1}$.

The final part is essentially a restating of a result about divisors in lectures.

**Comment by artie2000** [comment]
<details>
<summary>(b)</summary>

$\deg\varphi=2$, and the map is ramified on

$\{(1:0:0),(1:\pm1:0),(0:0:1)\}$.

</details>
